v19.1/recommended-production-settings.md

```diff
@@ -39,19 +39,19 @@ Nodes should have sufficient CPU, RAM, network, and storage capacity to handle y
 
 #### CPU and memory
 
-- At a bare minimum, each node should have **2 GB of RAM and 2 vCPUs**.
-
-    More data, complex workloads, higher concurrency, and faster performance require additional resources; as a general rule of thumb, increase the number of vCPUs and additional memory to match the requirements of the workload.
+- At a bare minimum, each node should have **2 vCPUs and 2 GB of RAM**. More data, complex workloads, higher concurrency, and faster performance require additional resources.
 
     {{site.data.alerts.callout_danger}}
     Avoid "burstable" or "shared-core" virtual machines that limit the load on CPU resources.
     {{site.data.alerts.end}}
 
-- The ideal configuration is 4-16 vCPUs, 8-64 GB memory nodes (2-4 GB of memory per vCPU).
+- To optimize for throughput, use larger nodes, up to 16 vCPUs and 64 GB of RAM. Based on internal testing results, 16 vCPUs is the sweet spot for OLTP workloads.
+
+    To increase throughput further, add more nodes to the cluster instead of increasing node size; higher vCPUs will have NUMA](https://en.wikipedia.org/wiki/Non-uniform_memory_access)(non-uniform memory access) implications.
```

Comment:
I'd remove this note about NUMA. We haven't explored this but we have no reason to believe it's a major concern at this point. The immediate reason why going to 32 vCPUs and beyond loses efficiency is simple mutex contention.

Reply:
Also for the record, on AWS m5/c5 nodes, you can go up to 48 vCPUs in a single NUMA group, so NUMA isn't a concern on this platform until you get to 72 vCPUs.